ee/query-service/app/server.go

```diff
@@ -199,17 +202,61 @@ func (s *Server) createPrivateServer(apiHandler *api.APIHandler) (*http.Server,
 	}, nil
 }
 
+func getPATToken(r *http.Request) (string, error) {
+	patHeader := r.Header.Get("SIGNOZ-API-KEY")
+	if patHeader == "" {
+		return "", nil
+	}
+
+	authHeaderParts := strings.Fields(patHeader)
+	if len(authHeaderParts) != 2 || strings.ToLower(authHeaderParts[0]) != "bearer" {
```

Comment:
if we are not using standard "authorization" header, is there a reason to have "bearer" in the key format. Cant we just allow clients to pass token as is

Reply:
Makes sense. Thank you!